aws-attach-roles/iam-kubecost-s3-object-store-policy.json

```diff
@@ -1,21 +1,6 @@
 {
```

Comment:
@jessegoodier I see you change the file name and remove ThanosBucketName related resources here. Are there any reason for this changes ? It may require us to rewrite the instructions to match with the changes.